From a262d0f39a388fec139c6be5d9da206ed4f58e0c Mon Sep 17 00:00:00 2001
From: stephen hemminger <stephen@networkplumber.org>
Date: Wed, 12 Feb 2014 20:51:22 -0800
Subject: [PATCH 061/213] net: remove unnecessary return's
References: fate#318918
Git-commit: a262d0f39a388fec139c6be5d9da206ed4f58e0c
Patch-mainline: Merged into mainline

One of my pet coding style peeves is the practice of
adding extra return; at the end of function.
Kill several instances of this in network code.

I suppose some coccinelle wizardy could do this automatically.

Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
[ddiss@suse.de: restrict to ceph only]
Acked-by: David Disseldorp <ddiss@suse.de>

---
 net/ceph/osd_client.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/net/ceph/osd_client.c b/net/ceph/osd_client.c
index 5b7ef5b..760ed8f 100644
--- a/net/ceph/osd_client.c
+++ b/net/ceph/osd_client.c
@@ -2103,7 +2103,6 @@ bad:
 	pr_err("osdc handle_map corrupt msg\n");
 	ceph_msg_dump(msg);
 	up_write(&osdc->map_sem);
-	return;
 }
 
 /*
@@ -2302,7 +2301,6 @@ done_err:
 
 bad:
 	pr_err("osdc handle_watch_notify corrupt msg\n");
-	return;
 }
 
 /*
-- 
2.1.4

